src/components/Input/Input.js

```diff
@@ -0,0 +1,183 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+import sluggify from "../../utils/text";
+
+const FieldInputBox = styled.input`
+  height: 36px;
+  width: ${props => props.width || "10em"};
+  border-radius: 2px;
+  background-color: #ffffff;
+  border: solid 1px #8a8a8a;
+  padding-left: 18px;
+  line-height: 2.57;
+
+  .text__error-top & {
+    border-color: #d93a3a;
+  }
+  .text__error-left & {
+    border-color: #d93a3a;
+  }
+  &:hover {
+    border: 2px #026cdf solid;
+    padding-left: 17px;
+    padding-top: -1px;
+  }
+  &:-webkit-input-placeholder {
+    opacity: 0.35;
+    color: blue;
+  }
+  &:-moz-placeholder {
+    opacity: 0.35;
+    color: #37444a;
+  }
+
+  &:focus {
+    border: 1px #026cdf solid;
+    padding-left: 18px;
+    border-radius: 2px;
+    box-shadow: 0 0 5px 0 #026cdf;
+    background-color: #ffffff;
+    outline: none;
+  }
+  &:invalid.text--input-invalid {
+    height: 36px;
+    border-radius: 2px;
+    border: solid 1px #e33f3f;
+    background-color: black;
+  }
+  &:focus.text--input-invalid {
+    height: 36px;
+    border-radius: 2px;
+    border: solid 1px #e33f3f;
+    background-color: black;
+  }
+`;
+const ErrorBoxWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+const FieldInputWrapper = styled.div`
+  display: flex;
+  position: relative;
+  &.text--input-top {
+    flex-direction: column;
+    justify-content: left;
+  }
+  &.text--input-left {
+    flex-direction: row;
+  }
+  &.text--input-disabled {
+    pointer-events: none;
+  }
+`;
+
+const FieldInputText = styled.label`
+  font-size: 12px;
+  height: 13px;
+  line-height: 1.25;
+  text-align: left;
+  color: #37444a;
+  position: relative;
+
+  .text--input-disabled & {
+    opacity: 0.35;
+  }
+  .text--input-top & {
+    margin-bottom: 6px;
+  }
+  .text--input-left & {
+    margin-right: 11px;
+    margin-top: 10px;
+  }
+
+  &.text__error-top {
+    color: #d93a3a;
+    margin-top: 1px;
+
+    font-size: 12px;
+    font-weight: normal;
```

Comment:
I believe the properties with the value `normal` are applied by default so we should not have to add them here.